02-getting_started.Rmd

```diff
@@ -1,27 +1,58 @@
-# Getting Started 
-
-## Install git
-
-## Get the source code 
-
-## Compile and build
-
-### UNIX
-
-### Windows
-
-## Install dependencies
-
-### Linux
-
-### maxOS and OS X
-
-## Regenerate `configure`
-
-## Troubleshoot the build
-
-### Avoid recreating auto-generated files
-
-## Editors and Tools
-
-## Directory structure
+# Getting Started 
+
+These instructions cover how to install R in Windows. The tools required to build R and R packages in Windows are also discussed.
+
+## Installing R
+
+To install R in Windows follow these steps:
+
+1. Go to https://cran.r-project.org.
+
+2. Select `Download R for Windows`. You will be directed to a page which shows `Subdirectories` for installing R on Windows.
+
+3. Select `base` subdirectory. (Alternatively, you can select `install R for the first time`, it leads to the same page). The recent stable release distribution of R can be downloaded from here. 
+
+4. The distribution is distributed as an installer ($\ge$) `R-4.0.4-win.exe`.

```

Comment:
here we use `4.0.4` several times -- that makes the document a bit fragile for when the current version of R will change in the future.

I recommend at a minimum re-using a variable like `R_VERSION=4.0.4` and ideally trying to generate that variable programmatically (then we only need to re-compile the document in the future)

Reply:
thanks for noticing!